[DOCS] Add documentation for Calendar Bundle (#14555)
Sylius/Sylius: docs/components_and_bundles/bundles/SyliusCalendarBundle/index.rst

SyliusCalendarBundle
====================

Sylius Calendar simple component for PHP applications that abstracts date object creation,

> which the main purpose is to ease testing of date depended features.

Reviewer: ```suggestion
Its primary purpose is to ease testing of date and time dependent features.
```
